app/services/stats/calculations.rb

```diff
@@ -128,7 +128,7 @@ def self.calculate_bu(row)
 
     def self.calculate_percentage(value, total)
       if total == 0
-        0.0
+        nil
```

Comment:
@willmcb,  changing here seems working with xlsx,   initially, I was thinking of not changing here as calculations.rb is not only used by this report, but for monthly performance report too (from another side, it maybe a benefit),  and also it is sort of arguable thing for returning nil rather 0 for math calculation 🤔 

another way for this ticket is to change the function `to_csv` in `base_business_unit_performance_report.rb`  between line 102 and 105,  let me know what you think, sorry my suggestion maybe is too late 😳

Reply:
I have no strong opinions - only really that if this is okay and _works_ I'd rather just merge it 👍